<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>emg_visualizer_app</class>
 <widget class="QMainWindow" name="emg_visualizer_app">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>emg_visualizer_app</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="accessibleName">
         <string/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>2D Visualizer</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="maximumSize">
               <size>
                <width>450</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QGroupBox" name="groupBox_3">
                 <property name="title">
                  <string>Data I/O</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_2">
                  <item row="1" column="0">
                   <widget class="QPushButton" name="btn_connect2ddata">
                    <property name="maximumSize">
                     <size>
                      <width>70</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Connect</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLabel" name="lbl_statusshmstream">
                    <property name="text">
                     <string>Status: Disconnected</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0" colspan="2">
                   <widget class="QGroupBox" name="groupbox_emgipc_2">
                    <property name="title">
                     <string>IPC I/O (Shared Memory FD)</string>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                     <item>
                      <widget class="QPushButton" name="btn_selectshmfile">
                       <property name="maximumSize">
                        <size>
                         <width>70</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Choose</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="lbl_shmfile">
                       <property name="text">
                        <string>FD: </string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_2">
                 <property name="title">
                  <string>Buttworth Filter</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_9">
                  <item row="0" column="0">
                   <widget class="QGroupBox" name="groupBox_4">
                    <property name="title">
                     <string>Load Custom Filter</string>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <item>
                      <widget class="QPushButton" name="btn_selectfilterfile">
                       <property name="maximumSize">
                        <size>
                         <width>70</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Choose</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="lbl_filterfile">
                       <property name="text">
                        <string>File:</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QGroupBox" name="groupBox_5">
                    <property name="title">
                     <string>Apply Filter</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_11">
                     <item row="3" column="0">
                      <widget class="QRadioButton" name="rbtn_defaultfilter">
                       <property name="maximumSize">
                        <size>
                         <width>100</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Default</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0" rowspan="2">
                      <widget class="QRadioButton" name="rbtn_nofilter">
                       <property name="maximumSize">
                        <size>
                         <width>100</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>No Filter</string>
                       </property>
                      </widget>
                     </item>
                     <item row="4" column="0">
                      <widget class="QRadioButton" name="rbtn_customfilter">
                       <property name="maximumSize">
                        <size>
                         <width>100</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Custom</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1" rowspan="4">
                      <widget class="QLabel" name="lbl_buttercoefficients">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLabel" name="label">
                       <property name="text">
                        <string>Coefficients</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignCenter</set>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <widget class="QGroupBox" name="groupBox">
                   <property name="title">
                    <string>Plot</string>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <item>
                     <widget class="QPushButton" name="btn_2dplotstart">
                      <property name="text">
                       <string>Start</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="btn_2dplotstop">
                      <property name="text">
                       <string>Stop</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_emg">
         <attribute name="title">
          <string>EMG</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontallayout_emg">
            <item>
             <widget class="QFrame" name="frame">
              <property name="maximumSize">
               <size>
                <width>450</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_10">
               <item row="2" column="0">
                <widget class="QGroupBox" name="groupbox_emgplot">
                 <property name="title">
                  <string>Plot</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_4">
                  <item row="0" column="0">
                   <widget class="QPushButton" name="btn_plotstart">
                    <property name="text">
                     <string>Start</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QPushButton" name="btn_plotstop">
                    <property name="text">
                     <string>Stop</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QGroupBox" name="groupbox_emgdataio">
                 <property name="title">
                  <string>Data I/O</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_6">
                  <item row="1" column="3" colspan="2">
                   <widget class="QGroupBox" name="groupbox_emgipc">
                    <property name="title">
                     <string>IPC I/O (Message Queue)</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_5">
                     <item row="0" column="1">
                      <widget class="QLabel" name="lbl_mqfd">
                       <property name="text">
                        <string>FD: </string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="0">
                      <widget class="QPushButton" name="btn_choosemqfd">
                       <property name="maximumSize">
                        <size>
                         <width>65</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Choose</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QRadioButton" name="rbtn_emgserver">
                    <property name="maximumSize">
                     <size>
                      <width>15</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="3" colspan="2">
                   <widget class="QGroupBox" name="groupbox_emgserver">
                    <property name="title">
                     <string>EMG Server</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout">
                     <item row="0" column="2" colspan="3">
                      <widget class="QLineEdit" name="lineedit_ipaddress">
                       <property name="text">
                        <string/>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignCenter</set>
                       </property>
                       <property name="placeholderText">
                        <string>EMG Server IP Address</string>
                       </property>
                      </widget>
                     </item>
                     <item row="3" column="4">
                      <widget class="QPushButton" name="btn_sendservercmd">
                       <property name="text">
                        <string>Send</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLabel" name="lbl_ip">
                       <property name="maximumSize">
                        <size>
                         <width>25</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>IP:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="2" colspan="2">
                      <widget class="QLabel" name="lbl_serverterminal">
                       <property name="text">
                        <string>Server Terminal</string>
                       </property>
                      </widget>
                     </item>
                     <item row="4" column="2" colspan="3">
                      <widget class="QTextEdit" name="textedit_serverreplywindow">
                       <property name="readOnly">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item row="3" column="2" colspan="2">
                      <widget class="QLineEdit" name="lineedit_servercmd">
                       <property name="placeholderText">
                        <string>EMG Server Command</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QRadioButton" name="rbtn_emgipc">
                    <property name="maximumSize">
                     <size>
                      <width>15</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="3">
                   <widget class="QPushButton" name="btn_connectemgio">
                    <property name="maximumSize">
                     <size>
                      <width>75</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Connect</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="4">
                   <widget class="QLabel" name="lbl_statusemgio">
                    <property name="text">
                     <string>Status: Disconnected</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QGroupBox" name="groupbox_activeemgsensors">
                 <property name="title">
                  <string>Choose Active EMG Sensors</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_3">
                  <item row="1" column="1">
                   <widget class="QPushButton" name="btn_emg4">
                    <property name="text">
                     <string>EMG 4</string>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QPushButton" name="btn_emg5">
                    <property name="text">
                     <string>EMG 5</string>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QPushButton" name="btn_emg2">
                    <property name="text">
                     <string>EMG 2</string>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QPushButton" name="btn_emg1">
                    <property name="text">
                     <string>EMG 1</string>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                    <property name="checked">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QPushButton" name="btn_emg6">
                    <property name="text">
                     <string>EMG 6</string>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QPushButton" name="btn_emg7">
                    <property name="text">
                     <string>EMG 7</string>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QPushButton" name="btn_emg3">
                    <property name="text">
                     <string>EMG 3</string>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QPushButton" name="btn_emg8">
                    <property name="text">
                     <string>EMG 8</string>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
